lib/WooCommerce/TransactionalEmailHooks.php

```diff
@@ -48,7 +55,12 @@ public function useTemplateForWoocommerceEmails() {
   }
 
   private function getNewsletter() {
-    return Newsletter::findOne($this->settings->get(TransactionalEmails::SETTING_EMAIL_ID));
+    $newsletter = Newsletter::findOne($this->settings->get(TransactionalEmails::SETTING_EMAIL_ID));
+    if (!$newsletter) {
+      $this->transactionalEmails->init();
+      $this->settings->set('woocommerce.use_mailpoet_editor', false);
+    }
```

Comment:
Maybe we could add a comment here so that it is obvious that this is not an expected situation but the handling of an error caused by data that are missing. I bet someone would wonder why we turn the feature off.